app/views/shared/checks_list.erb

```diff
@@ -0,0 +1,15 @@
+<h3>HTTP checks</h3>
+
+<% checks.each do |check| %>
+  <div class="list-item <%= check.is_ok ? 'success' : 'fail' %>">
+    <a class="item-url" href="/check/<%= check.id %>"><%= check.project ? "[#{check.project.name}] #{check.name}" : check.name %></a>
```

Comment:
looks like you can use `check.name_with_project` instead of `check.project ? "[#{check.project.name}] #{check.name}" : check.name`